### PR TITLE
refactor fixtures structure

### PR DESCRIPTION
**Pull request by Pardiez**
Movidas las fixtures a una carpeta db/ y eliminada la carpeta data/ con los json que se usaron antes de conectar a la API

### CODE REVIEW

File: api/repository/tasks.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module Huertask
```
Comment (elmendalerenda) [inline api/repository/tasks.rb:1]: wtf? this must be a file from another PR :P Please review the changes before committing

Comment (Pardiez) [reply api/repository/tasks.rb:1]: :+1:
